app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    id: int
+    first_name: str
+    last_name: str
```

Comment:
add a space to the last row of the code

Reply:
An empty line at the end of the code is present in each file. git removes it by itself! Flake8 tests passed!